test/integration/examples/apiserver_test.go

```diff
@@ -141,7 +141,7 @@ func TestAggregatedAPIServer(t *testing.T) {
 	}()
 
 	// just use json because everyone speaks it
-	err = wait.PollImmediate(100*time.Millisecond, 10*time.Second, func() (done bool, err error) {
+	err = wait.PollImmediate(time.Second, time.Minute, func() (done bool, err error) {
```

Comment:
Why the change?

Reply:
It was flaking on my machine when it was under load (so I synced it with the values in the ETCD test).